testing/mod.ts

```diff
@@ -1,11 +1,46 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
 
 // Do not add imports in this file in order to be compatible with Node.
+import { red, green, white, bold } from "../colors/mod.ts";
+import diff, { DiffType } from "./vendor/diff.ts";
+import prettyFormat from "./vendor/pretty-format.js";
```

Comment:
As per the comment above, you cannot have imports in this file - as it is also used under Node - and Node won't be able to resolve. See https://github.com/denoland/deno/blob/1e837f3281413c7a9a3a27bb37b6829e3d2a85fa/tools/ts_library_builder/test.ts#L12

Ideally you could provide `/pretty-format/testing.ts` which would import `/testing/mod.ts` and re-export the same API but with pretty-formatting on the errors? 

Some day we'll re-write ts_library_builder in Deno and this restriction will go away, but for now let's have the pretty-asserts be opt-in.

Reply:
@ry Thanks :) Let me confirm should I move diff.ts too?